src/dotnet/commands/dotnet-new/FSharp_Console/$projectName$.fsproj

```diff
@@ -1,5 +1,4 @@
-﻿<Project ToolsVersion="15.0">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+﻿<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
@srivatsn it works as you say. Why i made it like that.

### too late for a big change

i am pretty sure asking bundling the whole `FSharp.NET.Sdk` is not going to be approve for merge so late, because too big change and is not MS code.

## like that i can update/fix the f# integration if needed

I dont want to embed package, because F# is not perfect like c#, so if i see a bug i can just fix that releasing a new version of `FSharp.NET.Sdk` (in templates there is `-beta-*`). 
It's nice for fix bugs, without waiting a new drop of cli. 
As an important note the F# compiler is **not** bundled inside cli, so i can updated the compiler releasing a new version of `FSharp.NET.Sdk`.

## added before Microsoft.NET.Sdk because like that works for Web.Sdk and NET.Sdk

Maybe just `Sdk="FSharp.NET.Sdk"` was ok too, but like that i need to duplicate for Web.Sdk and normal NET.Sdk. Again too much diff.
Maybe there was another way but i had only 1 day to understand how `Sdk` that work and workaround with a good enough solution, who can be updated later if bugs